src/utils/crypto/libsodium.ts

```diff
@@ -290,6 +291,17 @@ export async function deriveSensitiveKey(passphrase: string, salt: string) {
     }
 }
 
+export async function deriveInteractiveKey(passphrase: string, salt: string) {
```

Comment:
we should just rename `deriveIntermediateKey`  to `deriveInteractiveKey` 

that was bad naming choice